tests/QueueTest.php

```diff
@@ -182,6 +193,22 @@
     expect(pest()->valuestore->get('tenant_id'))->toBe('The current tenant id is: acme');
 });
 
+test('tenant connections do not persist after tenant jobs get processed', function() {
```

Comment:
If you removed the `static::revertToPreviousState()` call here https://github.com/archtechx/tenancy/pull/1128/files#diff-d2557c1097c40e15ec7fc42979d9d8133351c52d2c0e5ba6fb89d2e78caffbdaR72, would this test fail?

Reply:
Yeah, the tenant connection remains open, so this test fails